lib/zosconnectee-connector.js

```diff
@@ -24,23 +23,23 @@ var logger = require('../lib/logger');
 exports.initialize = function initializeDataSource(dataSource, callback) {
   var settings = dataSource.settings;
 
-    // Get the template file
+  // Get the template file
```

Comment:
The changes below this line (including this one) are just formatting/whitespace fixes made by `eslint --fix .`